File: app/src/main/java/com/ppjun/android/smzdm/mvp/model/entity/main/Response.java
package com.ppjun.android.smzdm.mvp.model.entity.main;

import com.google.gson.annotations.SerializedName;

public class Response<T> {

    @SerializedName("data")
    T data = null;

    public T getData() {
        return data;
    }


}

File: app/src/main/java/com/ppjun/android/smzdm/mvp/model/entity/Response.kt
package com.ppjun.android.smzdm.mvp.model.entity
import com.google.gson.annotations.SerializedName



data class Response<out T>(
		@SerializedName("error_code") val errorCode: String = "",
		@SerializedName("error_msg") val errorMsg: String = "",
		@SerializedName("smzdm_id") val smzdmId: String = "",
		@SerializedName("s") val s: String = "",
		@SerializedName("data") val data: T
)

